### PR TITLE
Change to Bulma framework

### PR DESCRIPTION
**Pull request by jvanbaarsen**
Because the old template was not looking that good, I decided to make the change to Bulma (and yeah new things are cool).

This change means having to go over ALL the views and update them, so sorry for the long diff..

Here are some screens:

**Server overview**
image

**Server services**
image

**App overview**
image

**Backup page**
image

**Login page**
image

Here is a cat eating stuff from a fork to make up for the long diff:

### CODE REVIEW

**Comment by jvanbaarsen** [comment]
Going to merge this in for the livestream. After-merge review would be appreciated :-)

File: app/assets/stylesheets/_apps.scss [hunk 1]
```diff
@@ -1,9 +1,8 @@
 .apps {
   margin-top: 20px;
-  @extend .row;
-
   .app {
-    @extend .col-md-4;
+    @extend .column;
+    @extend .is-4;
```
Comment (joshuajansen) [inline app/assets/stylesheets/_apps.scss]: 👍

File: app/views/servers/show/add_ssh.html.erb [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-<pre><%= @ssh_key %></pre>
-<div class="well text-center <%= "hidden" if @server.connected? %>" data-server-status="checking">
-  <p><i class="fa fa-circle-o-notch fa-spin"></i> Checking for the SSH key</p>
-  <p class="hint hidden"></p>
+<div class="columns servers">
+  <div class="column is-8 is-offset-2">
+    <h1 class="title">Setup for <%= @server.name %></h1>
+    <p>You need to add the following public SSH key to the "root" user of the server</p>
+    <div class="ssh-key highlight">
+      <button class="copy" data-clipboard-target="pre#sshkey">Copy</button>
```
Comment (joshuajansen) [inline app/views/servers/show/add_ssh.html.erb]: nice!

**Comment by joshuajansen** [comment]
Skimmed through the code. Looks good to me

**Pull request merged**